lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,169 @@
+;(function (define, accessibleModal) {
+    'use strict';
+
+    define(['jquery',
+            'backbone',
+            'underscore',
+            'underscore.string',
+            'text!templates/dashboard/course.underscore'],
+        function ($,
+                  Backbone,
+                  _,
+                  _s,
+                  courseTemplate) {
+
+            if (_.isUndefined(_s)) {
+                _s = _.str;
+            }
+
+            var CourseView = Backbone.View.extend({
+                tagName: 'li',
+                className: 'course-item',
+                template: _.template(courseTemplate),
+
+                events: {
+                    'click .action-more': 'toggleCourseActionsDropDown',
+                    'click .action-unenroll': 'unEnroll',
+                    'click .action-email-settings': 'emailSettings',
+                    'click #upgrade-to-verified': 'upgradeToVerified',
+                    'click #block-course-msg a[rel="leanModal"]': 'unRegisterBlockCourse'
+                },
+
+                initialize: function (options) {
+                    this.listenTo(options.tabbedView, 'rendered', this.rendered);
```

Comment:
@afeef so is the reason parent is passed through when initializing a CourseListView so that you can bind click events to a modal? Or is this.rendered doing something else?